analysis/lang/cro/stemmer_cro.go

```diff
@@ -0,0 +1,150 @@
+//  Copyright (c) 2018 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cro
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/blevesearch/bleve/analysis"
+	"github.com/blevesearch/bleve/registry"
+)
+
+const StemmerName = "stemmer_cro"
+
+var stemmingRules = []*regexp.Regexp{
```

Comment:
The stemmer is implemented using a complex set of regular expressions. Did you develop these yourself?  Or did they come from some other source?

Reply:
I did not come with those myself. I've linked the original source here https://github.com/blevesearch/bleve/pull/1517/files#diff-5bac4c8b0a762d326c6c1fe0a4052418fb98fc6936a715374df526dc2fc418b6R25, but now that you've mentioned I should make it a bit clearer in the comments where this originated from.